topics/dev/tutorials/tool-integration/slides.html

```diff
@@ -811,6 +812,20 @@
 ---
 
 ### `tests`
+#### Comparison Methods: checking the stdout/stderr
+
+```xml
+<assert_stdout>
+    <has_text text="Step 1... determine cutoff point" />
+    <has_text text="Step 2... estimate parameters of null distribution" />
+</assert_stdout>
+```
+
+.footnote[[Complete documentation](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents)]
```

Comment:
@lecorguille The link should be https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-assert-stdout , can you fx it? I'm away from a computer ATM.